thorlcr/activities/result/thresult.cpp

```diff
@@ -77,7 +77,6 @@ class CResultActivityMaster : public CMasterActivity
         if (!results && !abortSoon)
         {
             ActPrintLog("WARNING: no results");
-            IHThorRemoteResultArg *helper = (IHThorRemoteResultArg *)queryHelper();
```

Comment:
Makes the comment on the line below a bit less meaningful.

Reply:
yes, the exception is good enough, I'll remove the comments.